e2e-tests/ukef/cypress/e2e/journeys/portal/test-data/AIN-deal-unissued-facilities/dealReadyToSubmit.js

```diff
@@ -0,0 +1,24 @@
+const dealThatJustNeedsDates = require('./dealThatJustNeedsDates.json');
+const { today } = require('../../../../../../../e2e-fixtures/dateConstants');
+
+export const generateAinDealUnissuedFacilitiesWithDates = () => {
+  const nowDay = today.dayLong;
+  const nowMonth = today.monthLong;
+  const nowYear = today.year;
+
+  const deal = { ...dealThatJustNeedsDates };
+
+  deal.submissionDetails['supplyContractConversionDate-day'] = nowDay;
+  deal.submissionDetails['supplyContractConversionDate-month'] = nowMonth;
+  deal.submissionDetails['supplyContractConversionDate-year'] = nowYear;
+
+  deal.mockFacilities[0]['conversionRateDate-day'] = nowDay;
+  deal.mockFacilities[0]['conversionRateDate-month'] = nowMonth;
+  deal.mockFacilities[0]['conversionRateDate-year'] = nowYear;
+
+  deal.mockFacilities[1]['conversionRateDate-day'] = nowDay;
+  deal.mockFacilities[1]['conversionRateDate-month'] = nowMonth;
+  deal.mockFacilities[1]['conversionRateDate-year'] = nowYear;
+
+  return deal;
+};
```

Comment:
Do we need to create this? I can see 7 repeated instances of this:

```js
['supplyContractConversionDate-day'] = nowDay;
```

Reply:
@ttbarnes  There was no mock data for AIN unissued facilities so I created these files. I think if we want to change this approach that needs to be done in a separate ticket. My ticket was only for adding an e2e test and I generally followed the same approach with the rest of the e2e tests.

Reply:
OK makes sense, thanks